backend/grants/admin.py

```diff
@@ -95,6 +96,9 @@ def dehydrate_submission_admin_link(self, obj):
             ]
         )
 
+    def dehydrate_grant_admin_link(self, obj: Grant):
+        return f"https://admin.pycon.it/admin/grants/grant/?q={'+'.join(obj.full_name.split(' '))}"
```

Comment:
weird link 😂 why not the detail page?

Reply:
because in the change page you will not see the email clearly.

Reply:
probably wee need to discuss how to send them, they are too many :D